internal/web_server/web/views/components/terms_of_service.templ

```diff
@@ -0,0 +1,74 @@
+						<li>Du samtykker i å bruke Nyhetsjeger i samsvar med alle gjeldende lover og forskrifter.</li>
+						<li>Du er ansvarlig for alt innhold og aktivitet som oppstår under din konto.</li>
+						<li>Du samtykker i ikke å bruke Nyhetsjeger på en måte som kan skade, deaktivere, overbelaste eller forringe tjenesten eller på annen måte forstyrre andres bruk og glede av tjenesten.</li>
+					</ul>
+				</li>
+				<li class="my-5">
+					<h2 class="text-xl">Quiz, Poeng og Premier</h2>
+					<ul class="list-disc">
+						<li>Ved å delta i quiz på Nyhetsjeger kan du tjene poeng basert på din ytelse.</li>
+						<li>Dersom du vinner en konkurranse på Nyhetsjeger, vil du bli kontaktet via den registrerte e-postadressen eller mobilnummeret for å arrangere levering av premien.</li>
+						<li>Ved å delta i konkurranser på Nyhetsjeger samtykker du i å offentliggjøre ditt brukernavn i topplister.</li>
+					</ul>
+				</li>
+				<li class="my-5">
+					<h2 class="text-xl">Åndsverk og lisens</h2>
+					<ul class="list-disc">
+						<li>
+							Nyhetsjeger er et åpent kildeprosjekt underlagt vilkårene i GNU General Public License v3.0 ("GPL-3.0"). Du kan finne en kopi av lisensen på <a href="https://www.gnu.org/licenses/gpl-3.0.html">www.gnu.org/licenses/gpl-3.0.html</a>.
+						</li>
+						<li>Du forplikter deg til å overholde vilkårene i GPL-3.0 når du bruker, modifiserer eller distribuerer prosjektet</li>
+					</ul>
+				</li>
+				<li class="my-5">
+					<h2 class="text-xl">Endringer</h2>
+					<ul class="list-disc">
+						<li>
+							Vi forbeholder oss retten til å endre disse vilkårene og betingelsene når som helst etter eget skjønn. 
+							Eventuelle endringer vil bli kunngjort på nettstedet, og din fortsatte bruk av Nyhetsjeger 
+							etter slike endringer utgjør din aksept av de reviderte vilkårene og betingelsene.
+						</li>
+					</ul>
+				</li>
+				<li class="my-5">
+					<h2 class="text-xl">Ansvarsfraskrivelse</h2>
+					<ul class="list-disc">
+						<li>
+							Nyhetsjeger leveres "som det er" og "som tilgjengelig" uten noen garantier, 
+							verken uttrykt eller underforstått. Vi gir ingen garantier for at nyhetsjeger vil være 
+							feilfritt eller at tilgangen til tjenesten vil være konstant eller uforstyrret.
+						</li>
+					</ul>
+				</li>
+				<li class="my-5">
+					<h2 class="text-xl">Juridisk</h2>
+					<ul class="list-disc">
+						<li>
+							Disse vilkårene og betingelsene skal tolkes i samsvar med lovene i Norge, 
+							uten hensyn til konflikter av lovbestemmelser. Enhver tvist som oppstår under disse vilkårene og 
+							betingelsene skal være underlagt eksklusiv jurisdiksjon til domstolene i Norge.
+						</li>
+					</ul>
+				</li>
+			</ol>
+			<p>Vennligst kontakt oss på <a href="mailto:group04webshop@gmail.com">group04webshop@gmail.com</a> hvis du har spørsmål eller bekymringer angående disse vilkårene og betingelsene. Takk for at du bruker Nyhetsjeger!</p>
+}
```

Comment:
Do we commit this AI generated ToS. Think we can just put and empty TBD instead. since it is wrong on the licence thingy, and I Personally dont want any AI generated HTML pages in the code